guptahemant/insta-clone
# adding signup functionality

Adding Signup functionality.

Testing steps:

- navigate to /register url to check for the form.
- on signup , new user should be created which can be verified by checking users table.
- if user creationis successful ,user should be redirected to landing page.

Screenshot:

![Screenshot from 2022-04-19 19-38-23](https://user-images.githubusercontent.com/65488349/164023141-cb351c4a-84b3-4b88-b85b-0b1ce3a4b3dc.png)
